ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/Committer.scala

```diff
@@ -0,0 +1,66 @@
+package com.daml.ledger.participant.state.kvutils.committer
+
+import com.codahale.metrics
+import com.codahale.metrics.Timer
+import com.daml.ledger.participant.state.kvutils.DamlKvutils.{
+  DamlLogEntry,
+  DamlLogEntryId,
+  DamlStateKey,
+  DamlStateValue
+}
+import com.daml.ledger.participant.state.v1.ParticipantId
+import com.digitalasset.daml.lf.data.Time
+import org.slf4j.{Logger, LoggerFactory}
+
+trait Committer[Submission, PartialResult] {
```

Comment:
Explain high-level concept behind the 'Committer'. It is not obvious to me.

Reply:
added explanations.